config/core/resources/configuration.yaml

```diff
@@ -46,6 +55,12 @@ spec:
   scope: Namespaced
   subresources:
     status: {}
+  conversion:
+    strategy: Webhook
+    webhookClientConfig:
+      service:
+        name: webhook
+        namespace: knative-serving
```

Comment:
I'm wondering whether this is something we should just have the webhook add?

Reply:
Maybe - webhook would need to know the service that fronts it. Make an issue in `knative.dev/pkg` ?

Reply:
It knows it.  It's in the WebhookOptions attached to context.

Reply:
This would also affect the other webhooks configurations

Reply:
Captured here: https://github.com/knative/pkg/issues/1044